modules/detect.sh

```diff
@@ -13,6 +123,33 @@ function detect {
   fi
 }
 
+function detect {
+  case "$1" in
+    "")
+      checkargn $# 0
+      detector
+      ;;
+    "rpi")
+      checkargn $# 2
+      detectrpi "$2"
+      ;;
+    "arm")
+      checkargn $# 1
+      detectarm
+      ;;
+    "bluetooth")
+      checkargn $# 1
+      detectbluetooth
+      ;;
+    "wifi")
+      checkargn $# 1
+      detectwifi
+      ;;
+    "*")
```

Comment:
Think `"*")` should be `*)` without the double quotes.

Reply:
Nice catch. I think so too.